c/src/ml-api-remote-service.c

```diff
@@ -385,6 +390,10 @@ _mlrs_process_remote_service (nns_edge_data_h data_h)
           "Service num: %d", service_type);
       break;
   }
+
+done:
+  g_free (service_key);
+  g_free (service_str);
```

Comment:
IMHO, ```service_str``` could be freed just after ```_mlrs_get_service_type()```.

Reply:
Thanks, I added g_autofree macro for `service_str` and `service_key`.